lib/Distribution/Sle/16Latest.pm

```diff
@@ -0,0 +1,16 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The class represents current (i.e. latest) SLE 16 distribution and
+# provides access to its features.
+
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Distribution::Sle::16Latest;
+use parent Distribution::Opensuse::AgamaDevel;
+use strict;
+use warnings FATAL => 'all';
```

Comment:
and these two lines above maybe they are not needed either, right?

Reply:
yes, updated.

Reply:
I think strict , we cannot remove it, I see errors.

Reply:
put them back,  the VR: https://openqa.opensuse.org/tests/4516024#details